### DISCUSSION TITLE
Can someone clarify, is it possible to access images inside a page folder?

### DISCUSSION THREAD

**Discussion by geoffyuen** [datenstrom/community#229] [Ask a question] [answered]
Example: `/content/sample_page/` contains a `page.txt` and a bunch of images. Is it possible to display these images or do they have to be in the `/media/images/` folder?

Reason: I would like to have my pages more self-contained like Kirby or Stacey.

**Comment by markseuffert** [comment, marked as answer]
I am sorry, images inside a `/content/` folder are currently not supported.
